packages/camera/src/components/Capture/hooks.js

```diff
@@ -0,0 +1,156 @@
+import { useCallback, useMemo } from 'react';
+import { monkApi } from '@monkvision/corejs';
+
+import Constants from '../../const';
+import Actions from '../../actions';
+
+import getWebFileDataAsync from '../../utils/getWebFileDataAsync';
+import log from '../../utils/log';
+
+/**
+ * @param current
+ * @return {string}
+ */
+export function useTitle({ current }) {
+  return useMemo(() => {
+    if (!current.metadata) { return ''; }
+    const { label, id } = current.metadata;
+
+    if (Constants.PRODUCTION) { return label; }
+
+    return `${label} - ${id}`;
+  }, [current]);
+}
+
+/**
+ * @param camera
+ * @param current
+ * @param sights
+ * @return {function(): Promise<picture>}
+ */
+export function useTakePictureAsync({ camera, current, sights }) {
+  return useCallback(async () => {
+    log([`Awaiting picture to be taken...`]);
+    const picture = await camera.takePictureAsync();
+    log([`Camera 'takePictureAsync' has fulfilled with picture:`, picture]);
+
+    const payload = { id: current.id, picture };
+    sights.dispatch({ type: Actions.sights.SET_PICTURE, payload });
+
+    return picture;
+  }, [camera, current.id, sights]);
+}
+
```

Comment:
We shall keep in mind, to update this hook, so we can inject camera settings into image metadata

Reply:
Done.